### ISSUE TITLE
Visual design of site menu

### ISSUE THREAD

**Issue by noahmanger** [18F/openFEC-web-app#1153]
Usability testing revealed a couple key insights, and we're now ready to move this forward into visual design and then implementation. 

First, there was a pretty strong signal that users preferred Option B, where all pages are listed out:
![b - nav - 1](https://cloud.githubusercontent.com/assets/1696495/14222316/35b21ed2-f824-11e5-98a5-793591ed475e.png)

This view immediately gives them a clearer understanding of the types of pages contained under a given section, without having to guess what they mean. We do not currently have all of these pages ready, but we will eventually. 

Second, the "start exploring" button caused universal confusion. People do not know what to expect, so we need a better way of providing a link to the top-level data landing page. 

Third, "Candidate and committee profiles" was gave users the clearest understanding of the kind of information and presentation available through the search.

So the task here is to design how this works at desktop, tablet and mobile layouts. Once design is done, we will implement in #1155.

**Comment by jenniferthibault** [comment]
ping :bell:  @emileighoutlaw — I'm starting the visual design of this, which means we should look more seriously at finalizing the copy for:
- Each page title/nav item title 
  - What are the sub-nav items that are concise but specific and clear? Lookin' at you, "Most recent form 1's" which is convoluted and references unhelpful form numbers. In earlier IA mockups, I had used full titles instead of form numbers, like "Most recent Statements of Organization filed")
- What language should replace "start exploring" to help people get to the data visualizations/dashboards on the data landing page? 
  - This does not have to be a button, or it doesn't have to ONLY be a button. 
- Do you want to add or edit any language around the Profiles?

**Comment by jenniferthibault** [comment]
@emileighoutlaw this is a work in progress, but wanted to show how things are moving around so that you can start mingling design & content thoughts.

![screen shot 2016-04-08 at 12 35 14 pm](https://cloud.githubusercontent.com/assets/11636908/14390702/67d511be-fd86-11e5-8166-2d016de8aeec.png)

One design decision I've altered, is that if we're grouping the button (previously, "start exploring") with the dashboard stuff, I think it needs to link people to the data dashboard section of the data landing page. So that's my intent with "get started". Seeing candidate search in here and all the data table options, I'm thinking it's less and less important to bring people to the top of the data landing page, esp if the search is enabled in the nav. 

I'm mostly playing with small icon designs on this now, so @noahmanger / @brethauer feel free to add any thoughts that strike you.

**Comment by jenniferthibault** [comment]
I should add that I'm also working on mobile mockups!

**Comment by jenniferthibault** [comment]
Ok! Some small line weight and typography changes to the desktop view. I'm going to continue making icon tweaks next week so that this can move forward.

![desktop - nav 1](https://cloud.githubusercontent.com/assets/11636908/14396155/002468b2-fda4-11e5-8590-50bf40e3cadb.png)

**Mobile mockups:**
Questions on my mind:
- do we need a direct link to get people back to the home nav screen if they're three clicks deep?
- do we need a "close" button on the open mobile nav? Right now, the mechanism to close is to click on the menu icon in the nav 

![screen shot 2016-04-08 at 4 09 17 pm](https://cloud.githubusercontent.com/assets/11636908/14396216/5109d406-fda4-11e5-8619-0c487e6b923c.png)

![screen shot 2016-04-08 at 4 11 17 pm](https://cloud.githubusercontent.com/assets/11636908/14396257/93c0a2fc-fda4-11e5-8511-9df1156e4f7e.png)

**Comment by emileighoutlaw** [comment]
I've been lost in meetings but haven't forgotten this! I'm on it today!

**Comment by emileighoutlaw** [comment]
Warning— long post of copy notes. (Please let me know if there's a better place to leave these!) 

Also, asking for y'alls thoughts on every line seems insane. So I put forth the below as my recommendations, with the caveat that I'd love to hear if you disagree on anything.

### Interactive overviews 
- Lowercase `o` for `overviews`
- `Election fundraising and spending` (as the microcopy)
    - _Reasoning: We don't want it to look like two options of which you select one from the menu._
- Instead of a button, hyperlink the text: `Election fundraising and spending`
    - _Reasoning: That way it would be the same as the way we have our "advanced" section set up. It seems a little odd to me to have an inconsistent experience in the same menu._

### Elections
- Let's retitle this `Location`
   - _Reasoning: I have a hunch "location" is a faster clue for users to spot and process when they want to find information by geography._ 
- Let's make the microcopy one line: `Search by state, district or ZIP code`
    - _Reasoning: Let's not split things onto two lines (which feels like we're saying ZIP is a better search than the smaller-text options). Once it's all on one line,  "ZIP code" will stand out no matter what, because it's all caps. So we order in descending size, leading with "state" (which I believe is the next most likely thing folks will search, after ZIP)._

### Advanced
- Let's call this either `Advanced data` or just `Advanced`
    - _Reasoning: "Datatables" is a weird (and not real) word. I like `Advanced` best, but I'm totally cool with `Advanced data` too._
- Forms: Let's use `Most recent Statements of Candidacy (Form 2)` and `Most recent Statements of Organization (Form 1)`.
    - _Reasoning: We should be grammatically okay to say `Form 2` rather than `Form 2s`, since the parenthetical is also a clarification._

### Design question
Will the first column ever have icons, the way that `Advanced` does? I'm just seeing how visually alluring that advanced information looks in this current design. I wouldn't want folks to accidentally be drawn away from the more novice sections.

**Comment by jenniferthibault** [comment]
Alrighty, here's the above changes implemented after Emileigh and I talked through it:

![desktop - nav 1 copy](https://cloud.githubusercontent.com/assets/11636908/14476911/9e1288ce-00d8-11e6-9afc-5f2b24a1ba18.png)


@emileighoutlaw one more content question I think we both missed:
`Form 5 filers only` (under `Report summaries` ) — This feels like it needs a plain language translation like we did with form 1 & 2's

@noahmanger : Emileigh and I had a question about the links for `Fundraising` `Spending` and the button for `Get started`. We interpreted your mockup to mean that the Fundraising and Spending links take you directly to those sections of the data landing page. But where does the button take you? It seems like it would also take you to that section of the page. We like the positive action it brings to the overviews, but want to make sure we're understanding your intent before we assign it different actions.

**Comment by noahmanger** [comment]
This is looking rad. My only intent with that button was to take people to the data landing page (and I imagined it taking them to the top of it). I'm not sure how I feel about having it under "interactive overviews". On the one hand I agree that it's nice to have a call to action there, and in many ways it is the best way to get started with the interactive overviews (great term by the way). On the other, I feel like it should be more clearly to the "top" section. Which is to say...I'm not sure.

**Comment by jenniferthibault** [comment]
Oh IIIIIIIIIIII see where I goofed it now. You had originally grouped the "start exploring" button with the `Profiles` section, which made sense as to why it brought you to the top of the data landing page.  I mistakenly regrouped the button with the Overviews. 

Here's one way we could reconcile this by adding a second call to action button. My instinct is to keep the Interactive overviews first on the list, so I've kept that, but I'm not sure if that will introduce confusion because the data landing page orders Profile search first.

![desktop - nav 1 copy 2](https://cloud.githubusercontent.com/assets/11636908/14496998/3f6e098e-0163-11e6-9ec1-6b23a3c43c98.png)

I don't think `Explore` is very meaningful, but was trying to not repeat `Get started`. Not sure if that's important or not.

**Comment by jenniferthibault** [comment]
Ugh. I think I just talked myself out of that option, and may have over-prioritized the dashboards because that's what we've been researching the past few weeks. Looking for a specific candidate/committee should still be the easiest option for folks to find. Sorry for the stream of consciousness.

I **do** think that the nav should reflect the order on the page, and I have an option for how we reorder it and show that each section is clickable, which I think is an improvement overall:

![desktop - nav 1 copy 2](https://cloud.githubusercontent.com/assets/11636908/14497782/fddcb40e-0165-11e6-971f-9bf4985cebd6.png)

**Comment by emileighoutlaw** [comment]
I'm really enjoying that last mock-up, Jen. For Form 5 filers, how about:

`Independent expenditure (Form 5) filers`

**Comment by jenniferthibault** [comment]
I dig it Emileigh! Let's go with that. 

@noahmanger I think this last mockup is ready to move forward with for the first round (including the last copy edit above). Closing out to move to implementation.
